.travis_scripts/create_feedstocks

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 set -e
+set -x
```

Comment:
Just as a friendly note, we should probably disable this at least temporarily before the token. Fortunately Travis CI is pretty diligent about scrubbing the logs so this info doesn't show up. In any event, added PR ( https://github.com/conda-forge/staged-recipes/pull/5260 ) to keep it hidden in the future.